pootle/static/js/browser/components/TopContributorsTable.js

```diff
@@ -65,7 +65,7 @@ const TopContributorsTable = React.createClass({
       );
     }
     return (
-      <div>
+      <div className="bd">
```

Comment:
Maybe you can land a fix for this, but displaying the right font size for the header.